test/unit/server-config-configmap.bats

```diff
@@ -61,6 +61,66 @@ load _helpers
   [ "${actual}" = "true" ]
 }
 
+#--------------------------------------------------------------------
+# global.metrics.enabled & ui.enabled
+
+@test "server/ConfigMap: creates ui config with .global.metrics.enabled enabled & ui enabled" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-config-configmap.yaml  \
+      --set 'global.metrics.enabled=true' \
+      --set 'ui.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.data["ui-config.json"] | length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "server/ConfigMap: creates ui config with .ui.metrics.enabled enabled & ui enabled" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-config-configmap.yaml  \
+      --set 'ui.metrics.enabled=true' \
+      --set 'ui.enabled=true' \
+      . | tee /dev/stderr |
+      yq '.data["ui-config.json"] | length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "server/ConfigMap: creates ui config with .ui.metrics.enabled disabled & ui enabled" {
```

Comment:
```suggestion
@test "server/ConfigMap: does not create ui config with .ui.metrics.enabled disabled & ui enabled" {
```
I think I'm reading this correctly?

Reply:
haha..indeed you are!! good catch bud!

Reply:
Also a super super tiny nit you're using `ui.metrics.enabled enabled` and also `ui enabled`  can we just swap this all to `ui.metrics enabled and ui enabled` ?  the double enabled is not my fav